__docs__/wonder-blocks-birthday-picker/birthday-picker.stories.tsx

```diff
@@ -32,8 +32,10 @@ const meta: Meta<typeof BirthdayPicker> = {
                 excludeDecorators: true,
             },
         },
+    },
+    globals: {
```

Comment:
Updating how we configure the default viewport for a story using: https://storybook.js.org/docs/essentials/viewport#defining-the-viewport-for-a-story